packages/embark-blockchain-process/src/blockchain.js

```diff
@@ -216,7 +213,19 @@ Blockchain.prototype.run = function () {
         next();
       });
     },
-    function init(next) {
+    function initPluginClient(next) {
```

Comment:
> This skips internal node account management

Why skip this? I understand that Lightchain doesn't support it, but other node clients might do.

Do you mean that the client itself can decide whether or not to skip it?

Also, I don't see `initDevChain` in the custom part. Can't they have dev accounts?

Reply:
Skipping internal node account management **for plugins only** allows the plugin to manage accounts. Because we can't assume other blockchains will support the CLI way of managing accounts, we expose a method `createAndUnlockAccounts()` to do this in the plugin at the right time.

`initDevChain` is also skipped **for plugins only** in favour of exposing an `initChain` method in the base class. This lets the node set itself up as it needs. The plugin has access to the blockchain config so it can set itself up for development, testnet, mainnet, etc.